.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: nb-clean
+  name: nb-clean
+  entry: 'nb-clean clean'
+  require_serial: true
+  language: python
+  types_or: [jupyter]
```

Comment:
Given we've only one entry in the `types_or` array, does this need to be `types_or` or could we just uses `types` instead?

Reply:
`types` is implicitly `types_and` - a list of file types that must _all_ be matched. https://pre-commit.com/#filtering-files-with-types

In this instance it doesn't matter much, but I thought this slightly less risky (accidentally adding a constraint that then matches nothing).